network.md

```diff
@@ -226,6 +226,14 @@ Backpressure mechanism
 - syn-ack (receiver to sender) // ack the segment number received
 - ack (sender to receiver) // ack the segment number received
 
+## TCP Connection Termination
```

Comment:
Also, can we move it to the right spot? The cards are sorted in alphabetical order.

Reply:
OK

Reply:
Same remark

Reply:
Got it. I will fix it.